modules/fts/pages/fts-introduction.adoc

```diff
@@ -0,0 +1,124 @@
+
+[source,json]
+----
+"hits": [
+    {
+      "index": "trsample_623ab1fb8bfc1297_6ddbfb54",
+      "id": "landmark_21603",
+      "score": 2.1834097375254955,
+      "locations": {
+        "city": {
+          "paris": [
+            {
+              "pos": 1,
+              "start": 0,
+              "end": 5,
+              "array_positions": null
+            }
+          ]
+        },
+        "content": {
+          "dame": [
+            {
+              "pos": 23,
+              "start": 169,
+              "end": 173,
+              "array_positions": null
+            },
+...
+]
+----
+
+Examples of natural language support include:
+
+* _Language-aware_ searching; allowing users to search for, say, the word `beauties`, and additionally obtain results for `beauty` and `beautiful`.
+* _Scoring_ of results, according to relevancy; allowing users to obtain result-sets that only contain documents awarded the highest scores.
+This keeps result-sets manageably small, even when the total number of documents returned is extremely large.
+* _Fast indexes_, which support a wide range of possible text-searches.
+
+== Stages of Full text search query
+A Full Text Search query , once built at the client, can be targeted to any server in the Couchbase cluster hosting the search service. 
+
+Here are the stages it goes through:
+
+1) The server that the client targets the search request to, assumes the role of the orchestrator or the coordinating node once it receives the external request.
+
+2) The coordinating node first looks up the index (making sure it exists).
+
+3) The coordinating node obtains the “plan” that the index was deployed with. The plan contains details on how many partitions the index was split into and all the servers’ information where any of these partitions reside.
+
+4) The coordinating node sets up a unique list of servers that it needs to dispatch an “internal” request to. A server in the Couchbase cluster is eligible if and only if it hosts a partition belonging to the index under consideration.
+
+5) Once the internal requests have been dispatched by the coordinating node to each of the servers, it’ll wait to hear back from them. Simultaneously, if any of the index’s partitions are resident on the coordinating node – search requests are dispatched to each of those partitions as well (disk-bound).
+
+6) Those servers in the cluster that receive the “internal” request from the coordinating node will forward it to each of the index partitions they host (disk-bound).
+
+7) Separate search requests that are dispatched concurrently to all index partitions resident within a server, and the server waits to hear back from them.
+
+8) Once the server hears back from all the partitions it hosts, it merges the results obtained from each of the partitions before packaging them into a response and shipping it back to the coordinating node.
+
+So to sum it up, the coordinating node waits for responses from,
```

Comment:
"So to sum it up" feels too casual, appropriate in a blog post, but not formal docs...

Reply:
Update: Remove "So to sum it up"